src/main/java/redis/clients/jedis/BinaryJedis.java

```diff
@@ -43,7 +43,7 @@
 public class BinaryJedis implements BasicCommands, BinaryJedisCommands, MultiKeyBinaryCommands,
```

Comment:
No. Two constructors that are deprecated have their own reasons.
- `BinaryJedis(String)` it is due to deprecating `Connection(String)`.
- `BinaryJedis(JedisShardInfo)` it's still supported. But the overall code flow surrounding it is changed. I find `deprecated` easy to warn users in case it matters to them.